### PR TITLE
Timezone switching in UI

### PR DESCRIPTION
**Pull request by romalytvynenko**
Helps to fix 
https://github.com/RebelCode/rcmod-wp-bookings-front-ui/issues/15

### CODE REVIEW

File: src/Core/components/CfEddbkWizard.js [hunk 1]
```diff
@@ -124,6 +117,18 @@ export default function (store, bookingDataMap, TranslateCapable, MapBookingFiel
       }
     },
 
+    watch: {
+      service: {
+        deep: true,
+        immediate: true,
+        handler (newService, oldService) {
```
Comment (XedinUnknown) [inline src/Core/components/CfEddbkWizard.js]: Could you document what this does please?

File: src/Core/utils/makeMapStore.js [hunk 2]
```diff
@@ -50,6 +50,7 @@ export default function (deepGet) {
    * @return {*}
    */
   return (namespace, map) => {
+    map = JSON.parse(JSON.stringify(map))
```
Comment (XedinUnknown) [inline src/Core/utils/makeMapStore.js]: I've seen this before. Is this a way of deep-cloning an object? If so, isn't there are better way?

Comment (romalytvynenko) [reply src/Core/utils/makeMapStore.js]: I researched how to solve problem of adding observers by Vue. And this is the simplest and cleanest solution. Should I change it to something else?

Comment (XedinUnknown) [reply src/Core/utils/makeMapStore.js]: No, no, leave for now. Thanks.

File: src/Core/components/CfEddbkWizard.js (hunk 1 above)
Comment (XedinUnknown) [reply src/Core/components/CfEddbkWizard.js]: ...and the timezone will only change when the service changes *if* there was no service selected, or the selected timezone is the old service timezone. In other words, the timezone should not change automatically with the service if it was manually selected to be something else. Right?

Comment (romalytvynenko) [reply src/Core/components/CfEddbkWizard.js]: Yes, exactly

**Pull request merged**